include/glaze/json/read.hpp

```diff
@@ -394,6 +394,92 @@ namespace glz
          }
       };
 
+      template <str_view_t T>
+      struct from_json<T>{
+         template <auto Opts, class It, class End>
+         GLZ_ALWAYS_INLINE static void op(auto& value, is_context auto&& ctx, It&& it, End&& end) noexcept {
+            if (static_cast<bool>(ctx.error)) [[unlikely]] { return; }
+
+            if constexpr (!Opts.opening_handled) {
+               if constexpr (!Opts.ws_handled) {
+                  skip_ws<Opts>(ctx, it, end);
+               }
+
+               match<'"'>(ctx, it, end);
+            }
+
+            // overwrite portion
+            auto handle_escaped = [&]() {
+               switch (*it) {
+               case '"':
+               case '\\':
+               case '/':
+               case 'b':
+               case 'f':
+               case 'n':
+               case 'r':
+               case 't':
+               case 'u': {
```

Comment:
Should probably validate \u is followed by 4 hex digits when force_conformance == true. But I can make that change later.